apps/frontend/src/hooks/useGetNativeTokenPrice.ts

```diff
@@ -11,24 +13,24 @@ import { useCacheCall } from './useCacheCall';
 import { useCurrentChain } from './useChainStore';
 import { useDollarValue } from './useDollarValue';
 
-export const useGetNativeTokenPrice = () => {
+export const useGetNativeTokenPrice = (token?: 'BTC' | 'ETH') => {
```

Comment:
I wouldn't pass `token` as an optional param here but `chainId`. It will be more readable.